[BUG] dlpack/dlpack.h: No such file or directory when build libcudf

**Describe the bug**
[ 47%] Building CXX object CMakeFiles/cudf.dir/src/io/avro/avro.cpp.o
/opt/rh/devtoolset-7/root/usr/bin/c++ -DBOOST_NO_CXX14_CONSTEXPR -DCUDA_API_PER_THREAD_DEFAULT_STREAM -DCUDF_VERSION=0.19.0 -DJITIFY_PRINT_LOG=0 -DJITIFY_USE_CACHE -DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_OFF -DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_CUDA -DTHRUST_HOST_SYSTEM=THRUST_HOST_SYSTEM_CPP -Dcudf_EXPORTS -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/jitify-src -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/libcudacxx-src/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/src -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/arrow-src/cpp/src -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/arrow-src/cpp/src/generated -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/arrow-src/cpp/thirdparty/hadoop/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/arrow-src/cpp/thirdparty/flatbuffers/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/thrust-src -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/thrust-src/dependencies/cub -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/rmm-src/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/rmm-build/include -I/ssd1/chenrui/cudf-nvtx-on/cpp/build/_deps/spdlog-src/include -isystem /opt/rh/devtoolset-7/cmake/include -isystem /usr/local/cuda/include -O3 -DNDEBUG -fPIC -Wall -Werror -Wno-unknown-pragmas -Wno-error=deprecated-declarations -pthread -std=gnu++14 -o CMakeFiles/cudf.dir/src/io/avro/avro.cpp.o -c /ssd1/chenrui/cudf-nvtx-on/cpp/src/io/avro/avro.cpp
/ssd1/chenrui/cudf-nvtx-on/cpp/src/interop/dlpack.cpp:23:10: fatal error: dlpack/dlpack.h: No such file or directory
 #include <dlpack/dlpack.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/cudf.dir/build.make:865: CMakeFiles/cudf.dir/src/interop/dlpack.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....

**Steps/Code to reproduce bug**
scl enable devtoolset-7 "java/ci/build-in-docker.sh"

**Expected behavior**
build success

--- a/cpp/cmake/thirdparty/CUDF_GetDLPack.cmake
+++ b/cpp/cmake/thirdparty/CUDF_GetDLPack.cmake
@@ -16,7 +16,7 @@
 
 function(find_and_configure_dlpack VERSION)
     if(DLPACK_INCLUDE)
-        set(DLPACK_INCLUDE_DIR "${DLPACK_INCLUDE_DIR}" PARENT_SCOPE)
+        set(DLPACK_INCLUDE_DIR "${DLPACK_INCLUDE}" PARENT_SCOPE)
         return()
     endif()
     find_path(DLPACK_INCLUDE_DIR "dlpack"

